src/commands/debug_files/create_jvm_based_bundle.rs

```diff
@@ -0,0 +1,112 @@
+use std::fs;
+use std::path::{Path};
+use anyhow::{bail, Result};
+use clap::{Arg, ArgMatches, Command};
+use sentry::types::DebugId;
+use symbolic::debuginfo::sourcebundle::{SourceFileType};
+use uuid::{Uuid};
+use crate::api::Api;
+use crate::config::Config;
+use crate::utils::args::ArgExt;
+use crate::utils::file_search::ReleaseFileSearch;
+use crate::utils::file_upload::{FileUpload, SourceFile, UploadContext};
+use crate::utils::fs::{path_as_url};
+
+pub fn make_command(command: Command) -> Command {
+    command
+        .hide(true) // experimental for now
+        .about("Create a source bundle for the given JVM based source files (e.g. Java, Kotlin, ...)")
+        .org_arg()
+        .project_arg(false)
+        .arg(
+            Arg::new("path")
+                .value_name("PATH")
+                .required(true)
+                .help("The directory containing source files to bundle."),
+        )
+        .arg(
+            Arg::new("output")
+                // .short('o')
+                .long("output")
+                .value_name("PATH")
+                .required(true)
+                .help("The path to the output folder."),
+        )
+        .arg(
+            Arg::new("debug_id")
+                // .short('d')
```

Comment:
Do we want a short option here? Any preference if so?

Reply:
Same as above.